internal/crdReplicator/crdReplicator_operator_test.go

```diff
@@ -210,4 +223,55 @@ var _ = Describe("CRD Replicator Operator Tests", func() {
 			})
 		})
 	})
+
+	Context("replication tests by phase without networking", func() {
+		When("the peering phase is none", func() {
+			JustBeforeEach(func() { DisableNetworking() })
```

Comment:
Should be moved a level up, and converted to a BeforeEach block which mutates the fc object. Otherwise, you might risk creating flaky tests due to the switching between enabled and not enabled.

Reply:
For sure the JustBeforeEach block could be moved a level up and avoid calling it in the following When blocks. 
But we must stick with the JustBeforeEach since the foreigncluster itself is created in a JustBeforeEach.

Reply:
Yes, but the foreigncluster variable is defined in the BeforeEach block, and here you can just set the network parameter, while leaving to the existing JustBeforeEach block the actual creation (if I understood correctly your goal).

Reply:
Ok, using a BeforeEach we just change the Foreigncluster resource before it is created.  This way we have faster and cleaner way to have a Foreigncluster instance with networking disabled. Thanks!